### DISCUSSION TITLE
Performing subquery filters

### DISCUSSION THREAD

**Discussion by thomasa13** [directus/directus#10886] [Archive]
Hello everyone, 

Recently I have come accross the need to find elements of a table that don't have relationships (in a one to many). It is very easy in sql as you can do a subquery like this :
`SELECT * FROM TableA WHERE id NOT IN (SELECT TableA_id from TableB)`
However, I can't find anythin in the documentation allowing to perform this query from the api. Is there a way ?
There doesn't seem to be a way to access the related items of a many-to-one from the side that is related to multiple items

**Comment by rijkvanzanten** [comment]
There's no way to do a filter with a non-relational subquery like this
